staff/marti-malek/mailchimp/index.html

```diff
@@ -0,0 +1,113 @@
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. A vel minus maiores maxime quas, facere reiciendis architecto. Ad pariatur, atque praesentium adipisci tempora similique minima provident quidem corporis molestias tenetur.</p>
+            <button class="button">Sign Up Free</button>
+        </section>
+        <section class="center">
+            <div class="container">
+                <h3>What's New</h3>
+                <section class="one">
+                    <img src=".\styles\templates.jpg" alt="">
+                    <h4>New Feature:</h4>
+                    <h3>More Templates!</h3>
+                    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Molestiae, veniam? Deserunt fugit soluta nihil tenetur earum corporis quidem nobis. Repellendus, laboriosam magni id provident velit tempore hic modi alias quasi!</p>
+                    <a href="">Learn more</a>
+                    <img class="arrow" src=".\styles\arrow.png" alt="arrow">
+                </section>
+                <section class="two">
+                    <img src=".\styles\denim2.jpg" alt="">
+                    <h4>Customer Story:</h4>
+                    <h3>Raleigh Denim</h3>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Ullam fugit tenetur cumque, fugiat quas voluptas corporis, dolore eius dignissimos reiciendis iste totam rerum obcaecati, suscipit animi odio. Corrupti, ipsa dolore.</p>
+                    <a href="">Learn more</a>
+                    <img class="arrow" src=".\styles\arrow.png" alt="arrow">
+                </section>
+                <section class="three">
+                    <img src=".\styles\unfurlr-thumb.jpg" alt="">
+                    <h4>App Spotlight:</h4>
+                    <h3>Unfurlr</h3>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Similique molestias atque explicabo, magni, ipsam omnis in aperiam odit nulla distinctio quia facere esse libero suscipit quisquam laboriosam blanditiis accusantium dignissimos.</p>
+                    <a href="">Learn more</a>
+                    <img class="arrow" src=".\styles\arrow.png" alt="arrow">
+                </section>
+                <section class="four">
+                    <img src=".\styles\golden-monkeys.jpg" alt="">
+                    <h4>App Spotlight:</h4>
+                    <h3>Golden Monkeys</h3>
+                    <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Facere minus beatae officia! Sint doloremque atque unde dicta incidunt aliquid consequuntur quae nobis quo non, modi nulla eveniet quis similique architecto.</p>
+                    <a href="">Learn more</a>
+                    <img class="arrow" src=".\styles\arrow.png" alt="arrow">
+                </section>
+            </div>     
+        </section>
+    </main>
+    <footer>
+        <section class="one">
```

Comment:
as a navigation bar, should be nav>ul>li>a

Reply:
I don't think so, thx for the feedback though!